source/HardwareSerial.h

```diff
@@ -222,13 +205,46 @@ class HardwareSerial : public Stream
         // at least 16 bits in size
         return write((const uint8_t *) (&n), sizeof(int));
     }
+
+private:
+
+#if defined(_M_ARM)
+    const unsigned int m_txPinNo = 8;
```

Comment:
These variables are intended to contain the TX and RX pin numbers for whatever the current board actually is.  They are supposed to be non-board specific in name, and they are initialized with the actual pin numbers that are currently in use.  We can make this clearer, but I would like to do this in a future edit.